data_structures.py

```diff
@@ -0,0 +1,84 @@
+# 1) Write a function def solution(A) that, given an array A of N integers, returns the smallest positive integer (greater than 0) that does not occur in A.
+# For example, given A = [1, 3, 6, 4, 1, 2], the function should return 5.
+# Given A = [1, 2, 3], the function should return 4.
+# Given A = [−1, −3], the function should return 1.
+
+def smallest_positive_integer(A=[1, 3, 6, 4, 1, 2]):
```

Comment:
never leave the list or any other mutable objects as the default value for arguments